examples/ConstraintIB/eel2d/IBEELKinematics.cpp

```diff
@@ -149,22 +147,22 @@ IBEELKinematics::IBEELKinematics(const std::string& object_name,
         (*cit)->DefineConst("PI", pi);
 
         // Variables
-        (*cit)->DefineVar("T", d_parser_time);
-        (*cit)->DefineVar("t", d_parser_time);
+        (*cit)->DefineVar("T", &d_parser_time);
+        (*cit)->DefineVar("t", &d_parser_time);
         for (int d = 0; d < NDIM; ++d)
         {
             std::ostringstream stream;
             stream << d;
             const std::string postfix = stream.str();
-            (*cit)->DefineVar("X" + postfix, &(d_parser_posn[d]));
-            (*cit)->DefineVar("x" + postfix, &(d_parser_posn[d]));
-            (*cit)->DefineVar("X_" + postfix, &(d_parser_posn[d]));
-            (*cit)->DefineVar("x_" + postfix, &(d_parser_posn[d]));
-
-            (*cit)->DefineVar("N" + postfix, &(d_parser_normal[d]));
-            (*cit)->DefineVar("n" + postfix, &(d_parser_normal[d]));
-            (*cit)->DefineVar("N_" + postfix, &(d_parser_normal[d]));
-            (*cit)->DefineVar("n_" + postfix, &(d_parser_normal[d]));
+            (*cit)->DefineVar("X" + postfix, d_parser_posn.data()+d);
```

Comment:
Are you using the `clang-format` pre-commit hook (instructions [here](https://github.com/IBAMR/IBAMR/wiki/Contributing#pre-commit-hook))? I think that the style file in the library should result in `data() + d` instead of `data()+d`.

Not a big deal, but maintaining coding style consistent with `clang-format` can reduce code churn.

Reply:
My pre-commit hook does not work. Let me fix it.